lib/route.js

```diff
@@ -156,7 +156,7 @@ function buildRouting (options) {
 
     this.after((notHandledErr, done) => {
       const path = opts.url || opts.path
-      if (path === '/' && prefix.length > 0 && opts.method !== 'HEAD') {
+      if (path === '/' && prefix.length && opts.method !== 'HEAD') {
```

Comment:
is this faster?

Reply:
Yes, it should be

Reply:
I just run few benches here are the results: (case 1 is main case 2 is this branch)
```
case 1 x 793,311,168 ops/sec ±2.07% (90 runs sampled)
case 2 x 838,535,224 ops/sec ±0.84% (94 runs sampled)
case 1 x 813,989,092 ops/sec ±2.55% (92 runs sampled)
case 2 x 799,507,815 ops/sec ±1.45% (90 runs sampled)
case 1 x 812,507,713 ops/sec ±2.10% (86 runs sampled)
case 2 x 848,921,418 ops/sec ±1.11% (96 runs sampled)
case 1 x 854,137,890 ops/sec ±0.48% (96 runs sampled)
case 2 x 857,603,147 ops/sec ±0.17% (95 runs sampled)
case 1 x 859,320,269 ops/sec ±0.21% (94 runs sampled)
case 2 x 857,643,300 ops/sec ±0.65% (96 runs sampled)
```